perception/autoware_tensorrt_bevdet/config/cams_param.yaml

```diff
@@ -0,0 +1,93 @@
+cams:
```

Comment:
@cyn-liu 

Thank you for your PR!
Could you please use `/tf_static` instead of this config file?